hasher-matcher-actioner/webapp/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+
+export default function App() {
+  return (
+    <>
+      <nav class="navbar navbar-expand-md navbar-dark bg-dark">
+        <a class="navbar-brand" href="/">Hasher-Matcher-Actioner (HMA)</a>
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="/">Dashboard</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/">Get Metrics</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link disabled" href="/">Another Main Menu Item</a>
+          </li>
+        </ul>
+      </nav>
+      <main role="main" class="container">
+        <div>&nbsp;</div>
+        <div class="jumbotron mt-6">
+          <h1>HMA Dashboard</h1>
+          <p class="lead">Cool metrics:</p>
+          <MetricsTable />
+        </div>
+      </main>
+    </>
+  );
+}
+
+function MetricsTable() {
+  const [metrics, setMetrics] = useState(null);
+  useEffect(() => {
+    fetch("https://m1mx5hd9h0.execute-api.us-east-1.amazonaws.com/hma-api-function1")
```

Comment:
Would request this be moved out into a separate environment variable or config global. You can use an env var to pass the URL to the 'npm start' script and can then figure out how to pass that on to the javascripty bits. 

[1] https://create-react-app.dev/docs/adding-custom-environment-variables/#referencing-environment-variables-in-the-html

Reply:
Good recommendation on an eventual solution for this kind of thing.